snippets/logging-clusteradmin-access-logs-snip.adoc

```diff
@@ -0,0 +1,13 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+// * modules/logging-creating-new-group-cluster-admin-user-role.adoc
+//
+:_mod-docs-content-type: SNIPPET
+
+[IMPORTANT]
+====
+Querying application logs for multiple namespaces as a `cluster-admin` user, where the sum total of characters of all of the namespaces in the cluster is greater than 5120, results in the error `Parse error: input size too long (XXXX > 5120)`. For better control over access to logs in LokiStack, make the `cluster-admin` user a member of the `cluster-admin` group. If the `cluster-admin` group does not exist, create it and add the desired users to it.
+====
```

Comment:
@skrthomas This LGTM. Do you want any changes here?

Reply:
Lgtm!! Thank you @shreyasiddhartha and @stevsmit 🚀